grails-app/conf/Config.groovy

```diff
@@ -381,6 +381,120 @@ openboxes.dashboard.catalogsSummary.enabled = true
 // Default value for news summary
 openboxes.dashboard.newsSummary.newsItems = []
 
+openboxes {
+    tablero {
+        enabled = true
+        endpoints {
+            number {
+                inProgressPutaways {
+                    endpoint = "getInProgressPutaways"
+                    archived = false
+                    order = 4
+                }
+                inventoryByLotAndBin {
+                    endpoint = "getInventoryByLotAndBin"
+                    archived = false
+                    order = 1
+                }
+                inProgressShipments {
+                    endpoint = "getInProgressShipments"
+                    archived = false
+                    order = 3
+                }
+                receivingBin {
+                    endpoint = "getReceivingBin"
+                    archived = false
+                    order = 2
+                }
+                itemsInventoried {
+                    endpoint = "getItemsInventoried"
+                    archived = true
+                    order = 5
+                }
+                defaultBin {
+                    endpoint = "getDefaultBin"
+                    archived = true
+                    order = 6
+                }
+            }
+            graph {
+                inventorySummary {
+                    endpoint = "getInventorySummary"
+                    archived = false
+                    datalabel = true
+                    order = 1
+                    colors {
+                        labels {
+                            success = ["In stock"]
```

Comment:
I'd rather see this key'd by the status instead of the class. In fact, I think most of these config properties should be full objects rather than just key/value pairs. For example, the labels should either be a list of label objects 
```
labels { 
    [key: "inStock", label: "In Stock", class: "success"],
    [key: "aboveMaximum", label: "Above Maximum", class: "warning"]
}
```
or a key/object pair.
```
labels { 
inStock: [label: "In Stock", class: "success"],
aboveMaximum: [label: "Above Maximum", class: "warning"]
}
```
Do you have a preference? 

As much as possible we should confirm to a single standardized way of specifying these things.

Reply:
For the moment I think we can keep it as it is, and we can see letter how to add the keys

Reply:
@bamoure Sounds good. The other thing that's bothering me is the graph vs number nodes. It seems like we should just have a list of widgets and the widget defines it's type. Why do we need a parent node to group them?